src/service/Offers.ts

```diff
@@ -108,55 +109,52 @@ export const getOfferService = (
         ),
       );
 
-    const [
-      instance,
-      give,
-      want,
-      displayableGiveTemplate,
-      displayableWantTemplate,
-    ] = await Promise.all([
-      E(boardIdMarshaller).unserialize(instanceHandle),
-      convertProposals(giveTemplate),
-      convertProposals(wantTemplate),
-      makeProposalTemplateDisplayable(giveTemplate),
-      makeProposalTemplateDisplayable(wantTemplate),
-    ]);
+    const [give, want, displayableGiveTemplate, displayableWantTemplate] =
+      await Promise.all([
+        convertProposals(giveTemplate),
+        convertProposals(wantTemplate),
+        makeProposalTemplateDisplayable(giveTemplate),
+        makeProposalTemplateDisplayable(wantTemplate),
+      ]);
 
-    const offerForAction: OfferSpec = {
-      id,
-      invitationSpec: {
+    let invitationSpecToUse = invitationSpec;
+    let instanceBoardId = 'Continuing Invitation';
```

Comment:
is this string read by anything? seems like `undefined` is more appropriate for a continuing invitation because there's no instance to reference